arch/x86/kvm/pkvm/pkvm.c

```diff
@@ -1,9 +1,11 @@
 // SPDX-License-Identifier: GPL-2.0
 #include <linux/types.h>
 #include <linux/kvm_host.h>
+#include <asm/fpu/xcr.h>
 #include "init_finalize.h"
+#include "pkvm/lapic.h"
 #include "pkvm.h"
-#include "lapic.h"
+#include "x86.h"
```

Comment:
Maybe `<x86.h>` instead of `"x86.h"` (to differentiate KVM headers from pKVM ones)?

Reply:
Yeah, seems "../x86.h" and "../lapic.h" are better. In this case, to include the lapic.h header file in the arch/x86/kvm/pkvm/ folder, probably we don't need "pkvm/lapic.h" but just "lapic.h".